### PR TITLE
Add the possibility to see reactions and interact to an article from details

### PR DESCRIPTION
**Pull request by mohamedHedi1990**

### CODE REVIEW

File: webapp/src/main/webapp/groovy/news/webui/activity/UINewsFullActivity.gtmpl [hunk 1]
```diff
@@ -17,21 +24,94 @@
...
   def activity = uicomponent.getActivity();
-
   def news = uicomponent.getNews();
+  def activityDeletable = uicomponent.isActivityDeletable();
+  def activityCommentAndLikable = uicomponent.isActivityCommentAndLikable();
+
+%>
 
-if (activity && news) {
+<% if (activity && news) { //process if not null
```
Comment (thomasdelhomenie) [inline webapp/src/main/webapp/groovy/news/webui/activity/UINewsFullActivity.gtmpl]: Useless to close and reopen %> <%

File: webapp/src/main/webapp/groovy/news/webui/activity/UINewsFullActivity.gtmpl [hunk 2]
```diff
@@ -1,11 +1,18 @@
 <!-- Full details news Activity -->
 <%
+  import org.apache.commons.lang.ArrayUtils;
```
[the remark this answers is not in the document]
Comment (mohamedHedi1990) [reply webapp/src/main/webapp/groovy/news/webui/activity/UINewsFullActivity.gtmpl]: Fixed.

File: webapp/src/main/webapp/groovy/news/webui/activity/UINewsFullActivity.gtmpl [hunk 3]
```diff
@@ -1,11 +1,18 @@
...
   import org.exoplatform.portal.webui.util.Util;
+  import org.exoplatform.webui.form.UIFormTextAreaInput;
   import org.exoplatform.social.core.service.LinkProvider;
   import org.exoplatform.social.core.space.model.Space;
   import org.exoplatform.social.webui.Utils;
 
   import org.apache.commons.lang.StringEscapeUtils;
+  import org.apache.commons.lang.StringUtils;
```
[the remark this answers is not in the document]
Comment (mohamedHedi1990) [reply webapp/src/main/webapp/groovy/news/webui/activity/UINewsFullActivity.gtmpl]: Fixed.

File: webapp/src/main/webapp/groovy/news/webui/activity/UINewsFullActivity.gtmpl [hunk 4]
```diff
@@ -1,11 +1,18 @@
...
+
+  import org.htmlcleaner.HtmlCleaner;
```
[the remark this answers is not in the document]
Comment (mohamedHedi1990) [reply webapp/src/main/webapp/groovy/news/webui/activity/UINewsFullActivity.gtmpl]: Fixed.

File: webapp/src/main/webapp/groovy/news/webui/activity/UINewsFullActivity.gtmpl [hunk 5]
```diff
@@ -17,21 +24,94 @@
...
 
   def pcontext = Util.getPortalRequestContext();
-
   def labelActivityHasBeenDeleted = _ctx.appRes("UIActivity.label.Activity_Has_Been_Deleted");
-
   def activity = uicomponent.getActivity();
-
   def news = uicomponent.getNews();
+  def activityDeletable = uicomponent.isActivityDeletable();
+  def activityCommentAndLikable = uicomponent.isActivityCommentAndLikable();
```
[the remark this answers is not in the document]
Comment (mohamedHedi1990) [reply webapp/src/main/webapp/groovy/news/webui/activity/UINewsFullActivity.gtmpl]: Fixed.

File: webapp/src/main/webapp/groovy/news/webui/activity/UINewsFullActivity.gtmpl [hunk 6]
```diff
@@ -17,21 +24,94 @@
...
   import groovy.json.JsonOutput;
 
   def pcontext = Util.getPortalRequestContext();
-
   def labelActivityHasBeenDeleted = _ctx.appRes("UIActivity.label.Activity_Has_Been_Deleted");
-
   def activity = uicomponent.getActivity();
-
   def news = uicomponent.getNews();
+  def activityDeletable = uicomponent.isActivityDeletable();
```
[the remark this answers is not in the document]
Comment (mohamedHedi1990) [reply webapp/src/main/webapp/groovy/news/webui/activity/UINewsFullActivity.gtmpl]: Fixed.

File: webapp/src/main/webapp/groovy/news/webui/activity/UINewsFullActivity.gtmpl (hunk 1 above)
Comment (mohamedHedi1990) [reply webapp/src/main/webapp/groovy/news/webui/activity/UINewsFullActivity.gtmpl]: Fixed.

**Pull request merged**